aiotfm/utils/enum.py

```diff
@@ -89,4 +89,12 @@ class commu(Enum):
 	def __missing__(self, key):
 		if isinstance(key, int):
 			return 'int'
-		return 0
\ No newline at end of file
+		return 0
+
+
+class TradeState(Enum):
```

Comment:
A sixth state would be appreciated, which is basically ERROR (if a player leaves the room, disconnects, turns into shaman...). With the ones there it is enough anyway.

Reply:
I find it useless. When a trade has an error, the event `on_trade_error` is dispatched and the trade is deleted.

Reply:
That's true